src/Razor/src/Microsoft.CodeAnalysis.Razor/FilePathComparison.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.CodeAnalysis.Razor
+{
+    internal static class FilePathComparison
```

Comment:
??? We already have this YO

Reply:
No, we have `FilePathComparer` not `Comparison`

Reply:
bleh yeah makes sense

Reply:
Wait why do we need this at all? You can just use the comparer?

Reply:
Oh derp, ya. instead of `string.Equal(..., ..., FilePathComparison.Instance)` could just do `FilePathComparer.Instance.Equal(`